chart/templates/nginx.yaml

```diff
@@ -13,8 +13,6 @@ data:
         map $ssl_preread_server_name $selected_upstream {
             hostnames;  
             *.admin.uds.dev backend_admin;
-            keycloak.uds.dev backend_passthrough;
-            login.uds.dev backend_passthrough;
```

Comment:
If deleting these we could/should also delete the `upstream backend_passthrough` block below here.

Reply:
wilco